src/neptune/new/handler.py

```diff
@@ -518,10 +627,53 @@ def track_files(self, path: str, destination: str = None, wait: bool = False) ->
             attr = self._container.get_attribute(self._path)
             if not attr:
                 attr = Artifact(self._container, parse_path(self._path))
-
-            self._container.set_attribute(self._path, attr)
+                self._container.set_attribute(self._path, attr)
```

Comment:
Was it intended? Looks like some random accidentally added indent.

Reply:
It's intendent indent.
There is no need for re-set when it's defined.